src/components/noMatch/NoMatch.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
```

Comment:
what does noMatch mean?

for directory names, let's use all lowercase
`components/nomatch`